src/sedcli_main.c

```diff
@@ -101,6 +103,12 @@ static cli_option write_mbr_opts[] = {
 	{0}
 };
 
+static cli_option blocksid_opts[] = {
+	{'d', "device", "Device node e.g. /dev/nvme0n1", 1, "DEVICE", CLI_OPTION_REQUIRED},
+	{'r', "hwreset", "Clear events by setting Hardware Reset flag. Allowed values: 1/0", 1, "FMT", CLI_OPTION_REQUIRED},
```

Comment:
How about making it a Flag, rather than using it as a Format, taking values.

Reply:
Sure, I will change to Flag instead

Reply:
OK. If intention is to treat this option as integer accepting "0" or "1", please use appropriate flags (CLI_OPTION_RANGE_INT) and .min_value and max_value to enforce user input checks

Reply:
sure